PYME/LMVis/pipeline.py

```diff
@@ -799,6 +799,18 @@ def load_session(self, session_info):
 
         self.Rebuild()
 
+        # we need some way to calculate image bounds when loading a session
+        # - presumably this needs to happen after the Rebuild()?
+        # - there could also be an issue if the selectedDataSource only spans a subset of the image extent!
+        # - we therefore select 'FitResults' as datasource in the first attempt and fall back to the selected ds otherwise
```

Comment:
in most cases `extractFromMetadata` should still do it's thing, so it's unclear if we really need to have the FitResults fallback (if we ditched this, we could factor the bounds estimation out into a single function and avoid the duplication).

More generally, we might want to address bounds estimation to better cope with multiple source data, e.g., make it the super-set of the bounds of all datasets, rather than just the bounds of a "special" data set.

Reply:
Actually, in the short term, we can probably move this, complete with `FitResults` fallback, to a common `Pipeline._estimate_bounds()` that is shared between here and `OpenFile()`.

Reply:
Longer term, we should add an `.imgBounds` property (lazily computed and cached) to `tabular` datasets to mirror the one in the Image class (or PEP8 the image class and make it `ImageStack.image_bounds`, and replace `pipeline.image_bounds` with a property that computes the bounding box of all datasources.